XCUITests/BaseTestCase.swift

```diff
@@ -12,7 +12,7 @@ class BaseTestCase: XCTestCase {
 
     // These are used during setUp(). Change them prior to setUp() for the app to launch with different args,
     // or, use restart() to re-launch with custom args.
-    var launchArguments = [LaunchArguments.ClearProfile, LaunchArguments.SkipIntro, LaunchArguments.SkipWhatsNew]
+    var launchArguments = [LaunchArguments.ClearProfile, LaunchArguments.SkipIntro, LaunchArguments.SkipWhatsNew, LaunchArguments.StageServer]
```

Comment:
Instead of changing the set up for all the tests, it may can be set in the test suite itself, like for the [FirstRunTests](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FirstRunTourTests.swift#L6)

Reply:
I can do that, I just thought this one would be a reasonable default. Do we want any UI tests using production sync?